fixtures/local-mode-tests/tests/sw.test.ts

```diff
@@ -19,24 +15,19 @@ describe("worker", () => {
 
 		//since the script is invoked from the directory above, need to specify index.js is in src/
 		worker = await unstable_dev(path.resolve(__dirname, "..", "src", "sw.ts"), {
-			config: path.resolve(__dirname, "..", "src", "wrangler.sw.toml"),
+			config: path.resolve(__dirname, "..", "wrangler.sw.toml"),
 			experimental: {
 				disableExperimentalWarning: true,
-				disableDevRegistry: true,
 			},
 		});
-
-		resolveReadyPromise(undefined);
 	});
 
 	afterAll(async () => {
-		await readyPromise;
 		await worker.stop();
 		process.env.NODE_ENV = originalNodeEnv;
 	});
 
-	it.concurrent("renders", async () => {
```

Comment:
out of curiosity, was `concurrent` not working well?

Reply:
Removing it definitely seemed to improve stability